### DISCUSSION TITLE
javascript/array-and-array-like-object-and-iterable

### DISCUSSION THREAD

**Discussion by giscus** [howdy-mj/blog-v2#46] [Announcements]
# javascript/array-and-array-like-object-and-iterable

배열(Array)이란 여러 개의 값을 순차적으로 나열한 자료구조이며, 안의 값을 요소(Element)라 한다.

단순히 나열된 구조로 보이는 배열이 자바스크립트에서는 다소 독특한 데이터 같다. 자바스크립트에는 배열이라는 타입이 존재하지 않으며, 배열의 타입을 찍어보면...

https://www.howdy-mj.me/javascript/array-and-array-like-object-and-iterable

**Comment by nurugji** [comment]
정말 도움 많이되었습니다 :) 포스팅 하신거 다 읽어봐야겠어요!!
